week-6/Homework/mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,7 +1,32 @@
 // Implement the missing functionality according to the task description.
 // Hint: Remember which function you need to call when it's time to alarm.
+let timer;
+function setAlarm() {
+  setTime = document.getElementById("alarmSet");
+  let num = setTime.value;
+  if (num === "") {
+    return;
+  }
+  document.getElementById("alarmSet").value = "";
+  let count = document.getElementById("timeRemaining");
 
-function setAlarm() { }
+  if (timer) {
+    clearInterval(timer);
+    num < 10
+      ? (count.innerText = "Time Remaining: 00:0" + num)
+      : (count.innerText = "Time Remaining: 00:" + num);
+  }
```

Comment:
() for the conditional ternary  is missing here Altom. It needs to be fixed

Reply:
I think braces are optional. But might be a good practice to put them.